docs/intro/tutorial01.txt

```diff
@@ -52,9 +52,9 @@ code, then run the following command:
 
 .. console::
 
-   $ django-admin startproject mysite
+   $ django-admin startproject mysite djangotutorial
```

Comment:
Perhaps this was meant to be the following?

`django-admin startproject djangotutorial`

Reply:
Ah, never mind. My mistake in understanding how this command works.